packages/boost-histogram/test_boost_histogram.py

```diff
@@ -0,0 +1,42 @@
+from pyodide_build.testing import run_in_pyodide
+
+
+@run_in_pyodide(packages=["boost-histogram"])
+def test_boost_histogram():
+    import boost_histogram as bh
+    import unittest
+
+    class TestBasicHistogram(unittest.TestCase):
```

Comment:
I put similar code into test_wrapt.py, but as I'm looking at this I realized I'm not sure whether this actually runs the test. What does inheriting from `TestCase` do? Don't we also have to say to run the test somewhere?

Reply:
I'd guess that depends on how this is wrapped? I copied this out into a tmp.py file and ran `python -m unittest` on it, so I don't know if this actually can fail. I can break it then try the suggestion above.

Reply:
Yeah I think you need `unittest.main()` at the bottom or it won't do the test.

Reply:
I _think_ this fails if you intentionally break the tests, though I get a weird error (TypeError: can't assign to property "pyodide_fatal_error" on 14527384: not an object) locally.

Reply:
That uhh sounds like a bug. It would be nice to get a reproduction on that so we can fix it.

Reply:
It sounds like we somehow confused a char* or PyObject* with a JavaScript object.

Reply:
I ran:

```
pytest -k "firefox and histogram" packages/boost-histogram/test_boost_histogram.py
```

And got:

```
_______________________________________________________________________________________________________________________________ test_boost_histogram[firefox] _______________________________________________________________________________________________________________________________
Error running function in pyodide

TypeError: can't assign to property "pyodide_fatal_error" on 11549448: not an object

API.fatal_error@http://127.0.0.1:54639/pyodide.js:15:39523
Module.callPyObjectKwargs@http://127.0.0.1:54639/pyodide.js:15:7005
Module.callPyObject@http://127.0.0.1:54639/pyodide.js:15:7208
apply@http://127.0.0.1:54639/pyodide.js:15:14354
apply@http://127.0.0.1:54639/pyodide.js:15:13439
runPython@http://127.0.0.1:54639/pyodide.js:15:32222
run@http://127.0.0.1:54639/test.html:4:45
@http://127.0.0.1:54639/test.html:7:40
@http://127.0.0.1:54639/test.html:25:15
@http://127.0.0.1:54639/test.html:27:8
```